buildpack.toml

```diff
@@ -6,7 +6,9 @@ name = "PHP"
 		".github/",
 		".gitignore",
 		".rspec_parallel",
-		"support/build/_*/",
+		"support/build/_conf/",
```

Comment:
@dzuelke This was merged without a changelog entry, since this repo doesn't have the check-changelog check:
https://github.com/heroku/heroku-buildpack-php/blob/main/CHANGELOG.md#v267---2025-05-15